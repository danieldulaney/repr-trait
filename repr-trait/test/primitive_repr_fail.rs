#[derive(repr_trait::PrimitiveRepr)]
enum Enum {
    A,
    B,
    C,
}

fn main() {

}
