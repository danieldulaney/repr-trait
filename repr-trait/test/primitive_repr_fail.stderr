error: proc-macro derive panicked
 --> test/primitive_repr_fail.rs:1:10
  |
1 | #[derive(repr_trait::PrimitiveRepr)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Can't derive PrimitiveRepr on a struct without repr(u*) or repr(i*)
